src/main.rs

```diff
@@ -365,15 +365,15 @@ fn generate_image_and_metadata(
         }
 
         combined_image
-            .save(format!("./{}/{}.png", output_path, image_name))
+            .save(format!("{}/{}.png", output_path, image_name))
```

Comment:
let file_path = format!("{}/{}.png", output_path, image_name);

....save(file_path.clone())

let mut file = File::create(file_path.clone()).unwrap();

Reply:
ufff my bad

Reply:
i will fix it in another PR